CONTRIBUTING.md

```diff
@@ -138,21 +139,37 @@ _Note: If you are not familiar with Python code, don't let yourself get distract
 
 ### Implementation Examples
 
-Have a look at this method: https://github.com/JOJ0/synadm/blob/107d34b38de71d6d21d78141e78a1b19d3dd5379/synadm/cli/user.py#L185
+Have a look at this method: https://github.com/JOJ0/synadm/blob/68749391d6a291d2fac229214f59924189c775ac/synadm/cli/user.py#L358-L369
 
-and this one: https://github.com/JOJ0/synadm/blob/107d34b38de71d6d21d78141e78a1b19d3dd5379/synadm/api.py#L80
+and this one: https://github.com/JOJ0/synadm/blob/68749391d6a291d2fac229214f59924189c775ac/synadm/api.py#L531-L545
 
 That's all it needs to implement command `synadm user details <user_id>`.
 
-And another example, this time using a POST based API endpoint. It implements command `synadm user password <user_id>`. This is the CLI-level method: https://github.com/JOJ0/synadm/blob/0af918fdeee40bc1d3df7b734a46e76bb31148b9/synadm/cli/user.py#L114
+And another example, this time using a POST based API endpoint. It implements command `synadm user password <user_id>`. This is the CLI-level method: https://github.com/JOJ0/synadm/blob/68749391d6a291d2fac229214f59924189c775ac/synadm/cli/user.py#L276-L301
+
+and again it needs a backend method in `api.py`: https://github.com/JOJ0/synadm/blob/68749391d6a291d2fac229214f59924189c775ac/synadm/api.py#L511-L529
+
+
+### Submitting data & URL encoding
+
+Since version 0.42 `synadm` encodes URL's in a central place - the `ApiRequest.query()` function in the `synadm.api` module.
```

Comment:
*checks commit logs since v0.41.1 to dev branch*

```
J0J0 Todos (7):
      Merge pull request #108 from JOJ0/default-draft-releases
      Update "Implement. Examples" chapter code-links
      Update "Helpers" chapter on CONTRIBUTING page
      Add URL-encoding chapter to CONTRIBUTING page
      Fix typos in URL-encoding chapter
      Apply suggestions from code review
      Apply suggestions from code review

Jackson Chen (2):
      make auto releases draft by default
      URL encoding in query function and refactor
```

Nope, doesn't seem like a minor release to me.

I think the commit should probably be included anyways alongside the commit itself, seems like a reasonable compromise of "when".

Reply:
I would've called it a minor release since it kind of "revolutionises" quite some internals and might even introduce bugs we couldn't think of yet.

But I see your point that it actually doesn't bring any new feature, it just fixes a bug and prevents future issues.